Angular/tsconfig.json

```diff
@@ -12,7 +12,7 @@
     "importHelpers": true,
     "target": "es2015",
     "typeRoots": [
-      "node_modules/@types"
+      "node_modules/@types",
```

Comment:
Doesn't this comma cause an issue?

Reply:
removed

Reply:
You can read up more about trailing commas here: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Trailing_commas